lib/rabbit.test.ts

```diff
@@ -1,7 +1,6 @@
 import test, { ExecutionContext } from "ava";
 import { connect } from "amqplib";
 import { ulid } from "ulid";
-import { AbortController } from "node-abort-controller";
```

Comment:
was this unused?

Reply:
It was a polyfill for node versions below 14, which this PR drops support for (since the new AWS sdk doesn't support it either)

Reply:
what's a polyfill?

Reply:
A polyfill adds support for newer features in older versions, so in v14 and below of node there was no AbortController, but the package node-abort-controller added support for it, so you could use the newer feature even with the older versions (but polyfills are typically less efficient).